src/AmbientSounds/ViewModels/SettingsViewModel.cs

```diff
@@ -87,6 +93,47 @@ public bool ResumeOnLaunchEnabled
             set => _userSettings.Set(UserSettingsConstants.ResumeOnLaunchKey, value);
         }
 
+        /// <summary>
+        /// The id of the current output device.
+        /// </summary>
+        public string CurrentOutputDeviceId
+        {
+            get => _currentOutputDeviceId;
+            set
+            {
+                if(_currentOutputDeviceId != value)
+                {
+                    _currentOutputDeviceId = value;
+                    _userSettings.Set(UserSettingsConstants.OutputAudioDeviceId, value);
+                }
+            }
+        }
+
+        /// <summary>
+        /// The current output device.
+        /// </summary>
+        public AudioDeviceDescriptor CurrentOutputDevice
+        {
+            get
+            {
+                var id = CurrentOutputDeviceId;
+                return OutputDevices.Where(x => x.DeviceId == id).FirstOrDefault();
```

Comment:
I think this might return null because of `FirstOrDefault()`, but the property claims to never return null (this viewmodel has an implicit `nullable enable` because it's in the .NET Standard 2.0 project). I think this needs to be clarified. Either the signature can be updated to `AudioDeviceDescriptor` or we return a stub object (aka `new AudioDeviceDescriptor { Id = string.Empty, Name = "--" }`)